near-sdk/compilation_tests/no_default.rs

```diff
@@ -0,0 +1,23 @@
+//! Smart contract with initialization function.
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::{near_bindgen, NoDefault};
+
+#[near_bindgen]
+#[derive(NoDefault, BorshDeserialize, BorshSerialize)]
```

Comment:
I am not sure whether `NotDefault` is more grammar-friendly.

Reply:
Yeah the phrasing makes it seems that by not adding `derive(Default)` means there is no default.  How about `EnsureNoDefault` or `FailingDefault`?

Reply:
Switched to `PanicOnDefault`